hazelcast/src/test/java/com/hazelcast/topic/TopicTest.java

```diff
@@ -627,4 +629,76 @@ public void onMessage(Message msg) {
         assertEquals(1000, stats.getPublishOperationCount());
         assertEquals(2000, stats.getReceiveOperationCount());
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testTopicMultiThreading() throws Exception {
```

Comment:
Seems a long running test (longer than 10 seconds) please annotate it with `NightlyTest ` as in [here](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/java/com/hazelcast/map/impl/InterceptorRegistryTest.java#L67)